src/components/common/FloatingActionButton/index.js

```diff
@@ -15,7 +15,19 @@ const buttonBackground = Platform.OS === 'android' && Platform.Version >= 21 ?
   TouchableNativeFeedback.Ripple(colors.rippleColor, true) : // eslint-disable-line new-cap
   null;
 
-class FloatingActionButton extends React.Component {
+type Props = {
+  onPress: func,
+  children: element,
+  position: string | number,
```

Comment:
please confirm the position might have number type value

Reply:
Use this instead `'center' | 'left' | 'right'`